src/frontend/components/Chat/SharedChatComponents/ChatInputBar/index.spec.tsx

```diff
@@ -1,10 +1,9 @@
 import React from 'react';
```

Comment:
commit message 
but if he
wants to join it and be able to write his own it has to authentificate himself.
A dedicated overlay component is add to fullfill this role. => but if he
wants to join it and be able to write his own he has to authenticate himself.
A dedicated overlay component is added to fulfill this role.

Reply:
message change, the new commit is now : [bbd802f18bd17164a6dbd213d6da78076372472e](https://github.com/openfun/marsha/pull/1276/commits/bbd802f18bd17164a6dbd213d6da78076372472e)